proctor-common/src/main/java/com/indeed/proctor/common/model/TestDefinition.java

```diff
@@ -55,6 +55,8 @@ public class TestDefinition {
 
     private boolean enableUnitlessAllocations;
 
+    private boolean silentExposureLogging;
```

Comment:
nit: Based on PROC-1413, should this be `isExposureLoggingSilent` or `enableSilentExposureLogging` or something else? (I'm terrible with names).

Reply:
updated